### ISSUE TITLE
23.03.2 Unable to update the add-ons provided by the NVDA Chinese Community Directory

### ISSUE THREAD

**Issue by cary-rowen** [josephsl/addonUpdater#17]
## Steps to reproduce
1. Install this version 0.1 add-on: https://github.com/viyf/NVDAUpdateMirror/releases/download/V0.1/VIYF-NVDAUpdateMirror-V0.1.nvda-addon
2. Check for updates using the Add-on Updater-23.03.2.
## Actual Performance
no update.
## Expected performance
Version 0.2 update found

## Additional Information
The update can be checked if using version 23.03 of the add-on updater.

**Comment by josephsl** [comment]
Hi, caused by protocol 2 modification to handle old and new community add-ons endpoints. I will do something about it for Add-on Updater 23.03.3 or 23.04 (likely the former). Thanks.
